configure.ac

```diff
@@ -523,7 +524,9 @@ fi
 
 dnl swig stuff
 if test -z "$ENABLE_SOS_TRUE"; then
-	which cython >/dev/null 2>&1 || AC_MSG_ERROR("cython not found")
```

Comment:
I think this is a left-over from when sos used to be a subproject of ovis. I don't think this check is needed anymore.